astropy/timeseries/periodograms/lombscargle/tests/test_lombscargle.py

```diff
@@ -186,10 +186,11 @@ def test_nterms_methods(
     else:
         P_expected = ls.power(frequency)
 
-        # don't use fast fft approximations here
+        # don't use fast fft approximations here if the SciPy is not available
```

Comment:
looks like this comment is out of sync now
```suggestion
        # don't use fast fft approximations here
```